locale/pt-br/blog/announcements/nodejs-certified-developer-program.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Os Parceiros da Fundação Node.js com a Fundação Linux no Novo Programa de Certificação Node.js
+date: 2017-1-26T12:00:00.000Z
+category: Comunicados
+slug: nodejs-certified-developer-program.md
+layout: blog-post.hbs
+---
+
+_Fundação Node.js a ser lançada
+fornecedor-neutro programa de certificação para a plataforma que mais cresce_
+
+**SAN FRANCISCO, Jan. 26, 2017** — [A Fundação Node.js](https://foundation.nodejs.org/), um liderado pela comunidade e
+apoiado pela indústria consórcio para avançar no desenvolvimento da plataforma Node.js,
+hoje anunciamos o desenvolvimento do programa inaugural de certificação Node.js
+destinado a estabelecer uma competência de linha de base no Node.js.
+
+Node.js é uma das linguagens de programação mais populares com mais de 4.5 milhões
+de usuários ativos por mês. Enquanto Node.js é cada vez mais difundido hoje com
+empresas de todos os tamanhos, organizações que desejam expandir o uso de Node.js
+muitas vezes lutam ao treinar desenvolvedores Java e recrutar
+novos talentos.
+
+“A fundação Node.js, com ajuda ded incríveis membros da comunidade e principais
+especialistas, está criando um programa abrangente de certificação que amplia o
+funil de conhecimento qualificado do Node.js disponível. Seja trabalhando em ambientes
+corporativos ou como consultores individuais, aqueles que se tornam desenvolvedores
+certificadores Node.js serão bem posicionados a começar em todo vapor como um desenvolvedor
+Node.js, possuindo habilidades que estão em alta demanda,” disse Tracy Hinds,
+gerente de comunidade educacional para a Fundação Node.js.
+
+O programa de desenvolvedor certificado Node.js, que está sendo desenvolvido com a contribuição dos principais
+especialistas Node.js e contribuidores, deverá estar disponível no segundo trimestre de 2017. O programa proverá um quadro geral
+de competência em Node.js,
+ajudando empresas a identificar rapidamente engenheiros Node.js qualificados, enquanto
+fornece desenvolvedores, empreiteiros e consultores com um caminho para diferenciar
+eles mesmos no mercado.
+
+Fundação Node.js é trabalhado em estreita colaboração com [A Fundação Linux](https://training.linuxfoundation.org/certification/why-certify-with-us) para criar o projeto
+e processos para administrar o programa. A Fundação Linux oferece uma
+casa neutra para executar programas de treinamento e certificação, obrigado por isso
+envolvimento próximo com a comunidade de código aberto. Isso oferece vários cursos
```

Comment:
"obrigado por isso envolvimento próximo com a comunidade de código aberto" is also missing some conjugation, but the translation looks weird